app/controllers/tournaments_controller.rb

```diff
@@ -1,22 +1,54 @@
+require 'chronic'
```

Comment:
You shouldn't have to require the gem as rails loads all the gems in the gem file on booting. You can always test it in the rails console.